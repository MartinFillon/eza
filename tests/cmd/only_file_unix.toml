bin.name = "eza"
args = "tests/itest --only-files"
