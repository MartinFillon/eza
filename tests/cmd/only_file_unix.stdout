a
b
c
d
e
f
g
h
i
image.jpg.img.c.rs.log.png
index.svg
j
k
l
m
n
o
p
q
